### PR TITLE
Blog post > New user manual

### PR DESCRIPTION
**Pull request by allanaaa**
New post inviting feedback and updating on the project. Image to come, and link to blog homepage.

### CODE REVIEW

**Comment by allanaaa** [comment]
Oh, do I need to add a link anywhere? Looks like blog.md maybe does it automatically.

File: _posts/2021-01-26-announcing-our-new-user-manual.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+We're looking for your feedback on every aspect: the structure, the text, the design, the images, and anything we've missed. We'd especially love to hear from you if you're a new user and are using this manual to install and run OpenRefine for the first time.
+
+We recommend that you start by [following along with the user manual](https://docs.openrefine.org/manual/installing#installing-or-upgrading) to [install or upgrade to the latest version of OpenRefine: version 3.4.1](https://github.com/OpenRefine/OpenRefine/releases/tag/3.4.1). 
+Then, try out some tasks alongside the relevant sections of the user manual. Don't try to read the whole thing - just focus on sections that are most important to you.
+
+If you'd like to share your thoughts:
+- Please fill out our feedback form: https://forms.gle/4KkoJFpSrWDMkrh6A. This helps us be sure we're hearing from a broad range of participants. Please fill out the form even if you don't have much to say - we'd love to know if you've taken a look regardless. There is space within the form to report specific requested changes and line edits, as well as give us your general impression.
```
Comment (wetneb) [inline _posts/2021-01-26-announcing-our-new-user-manual.md:22]:
```suggestion
- Please fill out [our feedback form](https://forms.gle/4KkoJFpSrWDMkrh6A). This helps us be sure we're hearing from a broad range of participants. Please fill out the form even if you don't have much to say - we'd love to know if you've taken a look regardless. There is space within the form to report specific requested changes and line edits, as well as give us your general impression.
```

File: _posts/2021-01-26-announcing-our-new-user-manual.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- You can also suggest edits directly on GitHub (use the "Edit this page" link at the end of each page of the [new docs](https://docs.openrefine.org/)). This will create a pull request with your suggestions; this requires a GitHub account.
+
+We're also asking people who have written tutorials, who give workshops, and who share links on their websites to update those links to point to [docs.openrefine.org](docs.openrefine.org).
```
Comment (wetneb) [inline _posts/2021-01-26-announcing-our-new-user-manual.md:25]:
```suggestion
We're also asking people who have written tutorials, who give workshops, and who share links on their websites to update those links to point to [docs.openrefine.org](https://docs.openrefine.org). 
```

File: _posts/2021-01-26-announcing-our-new-user-manual.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  author: allanaaa
+  title: "Announcing our new user manual"
+  category: Blog
+---
+
+The OpenRefine team is delighted to share [our new user manual](https://docs.openrefine.org/)!
+
+This reference covers every aspect of the tool, from installation to exporting a cleaned dataset.
+
+![A screenshot of the new user manual.](https://www.openrefine.org/images/2021-user-manual.png)
```
Comment (wetneb) [inline _posts/2021-01-26-announcing-our-new-user-manual.md:12]:
```suggestion
![A screenshot of the new user manual.](https://openrefine.org/images/2021-user-manual.png)
```

**Review by wetneb — APPROVED** [review]

**Pull request merged**